### PR TITLE
perf(www): Remove extraneous query fields in showcase-details query

### PR DESCRIPTION
**Pull request by tesseralis**
## Description

Remove extraneous query fields in site showcase query.

### CODE REVIEW

File: www/src/components/showcase-details.js [hunk 1]
```diff
@@ -155,7 +153,7 @@ function ShowcaseModal({ children, location }) {
       modalNextLink={
         <>
           <Img
-            key={nextSite.id}
+            key={nextSite.fields.slug}
```
Comment (tesseralis) [inline www/src/components/showcase-details.js]: are these keys even needed? They're not part of a list or anything.

Comment (fk) [reply www/src/components/showcase-details.js]: Good q … l👀king now (like I should have before I approved 😬).

Comment (pieh) [reply www/src/components/showcase-details.js]: I think keys are there to force components to unmount when cycling through showcase with prev/next buttons and without them there was some weirdness (old image staying visible or something like that)

Comment (fk) [reply www/src/components/showcase-details.js]: @pieh to the rescue! 💖

Comment (pieh) [reply www/src/components/showcase-details.js]: After thinking a bit more - this just seem like problem with `gatsby-image` that doesn't nicely handle case of props updates (i.e. it should blow away whatever state it had)? (if it still is the case, not sure if it is).

**Pull request merged**